{
  try {
    execute();
    endProcessing("end");
  }
 catch (  KettleJobException je) {
    System.out.println("A serious error occurred!" + Const.CR + je.getMessage());
  }
}
